config/clusteringress-istio.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: clusteringress-istio
+  namespace: knative-serving
```

Comment:
I wonder if we want to put this into `knative-networking` and run it as a separate SA

Reply:
Tried this out and its a pain to upgrade to as we currently create virtualservices (and istio gateway) in knative-serving. I think we should do this as a separate commit as a result since it will require some additional moving of resources.